src/triggers.js

```diff
@@ -266,6 +266,16 @@ export function getResponseObject(request, resolve, reject) {
       ) {
         return resolve(response);
       }
+      if (
```

Comment:
Why not the code below?
```
if (
  response &&
  typeof response === 'object' &&
  request.triggerName === Types.afterSave
) {
  return resolve(response);
}
```

Reply:
Work well with the other suggestion I update the PR